### DISCUSSION TITLE
Playlists not importing

### DISCUSSION THREAD

**Discussion by Bautar** [rern/rAudio#811] [Fixed Issues]
I've been a Rune Audio user for years and I've just switched one of my machines over to rAudio-1.  I have a large library loaded to the SD card & I'm using a Raspberry Pi Zero-W with a Pimoroni PHAT DAC board.  My problem is that after moving my .m3u playlist files into /var/lib/mpd/playlists and using the "Addons Import Playlist" function, the playlists do not show up in the rAudio-1 Library.  
Here is an example screenshot after import:
![rAudio-1 Playlist](https://user-images.githubusercontent.com/99979632/154711581-d5ee83c2-3ca0-4ea7-97ee-b18e78a0088d.PNG)

Checking /srv/http/data/playlists, the file is not present.  
All playlist files were edited in Notepad++ to covert EOL to LF.

Here is a clip of several lines from an m3u file which words in Rune Audio:
Localstorage/Carl Perkins/Original Sun Greatest Hits/01 Blue Suede Shoes.M4a
Localstorage/Carl Perkins/Original Sun Greatest Hits/02 Honey Don't.M4a

My new machine is fully functioning other than this last bug!  Looking forward to any help fixing this & then I'll convert my other machines.

**Comment by rern** [comment]
Changing directory name from `Localstorage` to `SD` has been added to import script.
Please try again.

**Comment by Bautar** [comment]
Thanks, Rern.  I wasn't able to confirm I had your latest import script or not (lack of technical competence on my side!), however based on your note I updated the directories to SD on my playlist files & imported them & now they work fine.  

Thank you!!

Side-note, I moved all 60 of my updated playlists to /var/lib/mpd/playlists and then used the Import function (again, not sure if it was your latest script).  The import worked, however each playlist seemed to build on the previous so when opening them, they contained thousands of titles from other playlists; perhaps the script didn't include a "close file" line between?  I worked around this by moving and importing the playlist files individually then deleting them from /var/lib/mpd/playlists.  So I'm all set but you may want to check this at some point.

rAudio-1 is FANTASTIC!  Thanks for all of the effort you've put into this!
